pipeline-steps/common.groovy

```diff
@@ -259,4 +259,37 @@ api_key = ${args.api_key}
   return pyrax_cfg
 }
 
+def prepareConfigs(Map args){
+  conditionalStage(
```

Comment:
conditionalStep can be used so that the block is still conditional, but doesn't create a nested stage.